cfpb/cfgov-refresh
# Ask audience pages

Adds Ask audience pages

## Additions

- Audience pages

## Changes

- Ask secondary nav generation

## Testing

- Check out branch
- Set up Ask if necessary
  ```
  export DEPLOY_ENVIRONMENT=build
  ./drop-db.sh
  ./create-mysql-db.sh
  mysql -uroot v1 < ~/Downloads/production_django.sql
  python cfgov/manage.py migrate
  python cfgov/manage.py runscript migrate_knowledgebase
  ```
- Navigate to [audience page](http://localhost:8000/ask-cfpb/audience-students/) and verify its contents
- Verify audience page links on landing and question pages

## Screenshots

![screencapture-localhost-8000-ask-cfpb-audience-students-1495554217081](https://cloud.githubusercontent.com/assets/778171/26362856/268f989a-3f94-11e7-9aa3-48c767aca1c4.png)


## Checklist
